static/templates/management.html

```diff
@@ -69,6 +76,38 @@
     </form>
 
 <script>
+        $(document).ready(function(){
+                $('#grid').css('visibility', 'hidden')      
+	})
+	function showSsids() {
+	    $('#login').css('visibility', 'hidden');      
+	    $('#grid').css('visibility', 'visible');     
+	}
+	function authenticate(){
+            var pw = $('#password').val();
+	    if ( pw.length < 8) {
+		alert("Password must be at least 8 characters long");
+	    } else {
+	        $.ajax({
+	          type: "POST",
+	          url: "/hashit",
+	          data: {Hash: pw}
+	        }).done(function (hashRet) {
+		    console.log("in ajax done.", hashRet);
+		    hash = JSON.parse(hashRet);
+		    console.log(hash)
+		    if (hash.HashMatch) {
+			showSsids();
+		    } else {
+			alert("Your password does not match, please try again");
```

Comment:
yes, as noted in my PR, the design/layout can/should be improved. Probably in a following PR